src/FlowRunTimeline.vue

```diff
@@ -390,11 +390,11 @@
       }
     })
 
-    watch(() => props.graphData, () => {
```

Comment:
This may not be getting called, when viewing the UI, the "slow feed data" option is no longer slow feeding nodes. I'm thinking updated data is not registering somewhere.

Reply:
looks like the watcher is being called and the layout is trying to update but the worker loses the layout setting 🤔

Reply:
Added a comment about how this was fixed. We good!